app/services/coronavirus_pages/timeline_entry_builder.rb

```diff
@@ -0,0 +1,29 @@
+class CoronavirusPages::TimelineEntryBuilder
+  def create_timeline_entries
+    if timeline_entries_from_yaml.any?
+      TimelineEntry.transaction do
+        coronavirus_page.timeline_entries.delete_all
+
+        timeline_entries_from_yaml.each do |entry|
+          TimelineEntry.create!(
+            heading: entry["heading"],
+            content: entry["paragraph"],
+            position: entry["position"],
```

Comment:
This isn't actually going to do anything because of: https://github.com/alphagov/collections-publisher/blob/3f8ee18910a0507a7824ae26b552de1d65942411/app/models/timeline_entry.rb#L5

It also looks like the yaml file doesn't have a position

Reply:
yep, absolutely right, rushed this in, should have checked, sorry